snirf_specification.md

```diff
@@ -496,9 +496,7 @@ for list of possible values.
 * **Type**:  integer
 * **Location**: `/nirs(i)/data(j)/measurementList(k)/dataTypeIndex`
 
-Data-type specific parameter indices. The data type index specifies additional data type specific parameters that are further elaborated by other fields in the probe structure, as detailed below. Note that the Time Domain and Diffuse Correlation Spectroscopy data types have two additional parameters and so the data type index must be a vector with 2 elements that index the additional parameters. One use of this parameter is as a 
```

Comment:
This is deleted because the same dataTypeIndex will map both parameters in probe (timeDelays and timeDelayWidths for TD, correlationTimeDelays and correlationTimeDelayWidths for DC).